scrolling-tabs.js

```diff
@@ -590,34 +588,36 @@
         }
 
         smv.slideMovableContainerToLeftPos();
+      } else {
+        stc.handleArrowsClassNames('left');
```

Comment:
Can you add a condition so this function is only called if `scrollingTabsActiveOffset` is set, so it doesn't affect other users.